src/Randomizer.SMZ3.Tracking/AutoTracking/USB2SNESConnector.cs

```diff
@@ -129,6 +129,9 @@ private void SendBinaryData(string address, byte[] data)
             });
 
             _client.Send(message);
+
+            await Task.Delay(TimeSpan.FromSeconds(0.05));
```

Comment:
Yeah, it's definitely kind of fucky, but you're right about it being to do with USB2SNES. It has to receive the PutAddress message first before I send the binary data, and I'm not sure if it was an issue with the two coming across at basically the same time caused the PutAddress message to be dropped sometimes or if it was coming across out of order, but whatever the case it caused USB2SNES to restart its websocket server. Tracker was able to reconnect, but still it was worrisome. It only seemed to happen once out of every 10-15 times, it could definitely be an issue if we were to use this for multiworld stuff.

For what it's worth, I tried to see if I got any sort of response from PutAddress to look for before sending the binary data, but it doesn't seem like USB2SNES gives any indicator that it's ready to receive the binary data. You just have to hope that it got the PutAddress message successfully or else it throws a little tantrum and restarts the websocket server.